app/src/models/item.ts

```diff
@@ -99,14 +129,15 @@ export class ItemModel {
       locations: manifestMetadataHash["Library Locations"]
         ? manifestMetadataHash["Library Locations"].join("<br>")
         : "",
-      subjects: manifestMetadataHash["Subjects"]
-        ? manifestMetadataHash["Subjects"].join("<br>")
-        : "",
+      subjects:
+        manifestMetadataHash["Subjects"].length > 2
+          ? manifestMetadataHash["Subjects"].join("<br>")
+          : "",
```

Comment:
Might be misunderstanding, but if there's only 1 subject, we'll set this to be empty